demo/demo.md

```diff
@@ -128,14 +73,44 @@ Depending on the element trigger that the popover element is assigned to, the sp
   <auro-button id="button10" slot="trigger">Popover w/additional space</auro-button>
 </auro-popover>
 
-<auro-popover for="button11" placement="bottom">
+<auro-popover for="button11" placement="bottom" addSpace>
```

Comment:
Content of this popover says its to show the default spacing - shouldn't the `addSpace` attribute not be present here?

Reply:
Typos. Good catch.